src/examples/java/com/amazonaws/crypto/examples/keyring/awskms/DiscoveryDecryptWithPreferredRegions.java

```diff
@@ -1,140 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Double check with June that she doesn't have doc changes for keyrings in Java that depend on this example

Reply:
Will do. My plan was to work with her as part of https://github.com/aws/aws-encryption-sdk-java/issues/225 since we'll need to finalize the examples (and their names/paths) later anyways.